stdlib/test/builtin/test_float_literal.mojo

```diff
@@ -54,6 +54,25 @@ def test_floor():
     assert_equal(FloatLiteral.__floor__(neg_inf), neg_inf)
 
 
+def test_round():
+    assert_equal(FloatLiteral.__round__(1.5), 2.0)
+    assert_equal(FloatLiteral.__round__(1.4), 1.0)
+    assert_equal(FloatLiteral.__round__(-1.5), -2.0)
+    assert_equal(FloatLiteral.__round__(-3.4), -3.0)
+    assert_equal(FloatLiteral.__round__(3.0), 3.0)
+    assert_equal(FloatLiteral.__round__(0.0), 0.0)
+
+    assert_true(FloatLiteral.__round__(nan).is_nan())
+    assert_true(FloatLiteral.__round__(neg_zero).is_neg_zero())
+    assert_equal(FloatLiteral.__round__(inf), inf)
+    assert_equal(FloatLiteral.__round__(neg_inf), neg_inf)
+
+    assert_equal(FloatLiteral.__round__(1.5, 1), 1.5)
+    assert_equal(FloatLiteral.__round__(1.123, 1), 1.1)
```

Comment:
Please add some negative input values here for completeness.

Reply:
Done: https://github.com/msaelices/mojo/commit/41966c8add2071c49bf5f7e95300be50a534e968